src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-bar_container">
+      <form
+        class="search-bar top-search-bar"
+        data-qa="big"
+      >
+        <input
```

Comment:
The `alt` attribute is missing on the input element. Although it's not required for input elements, if you had images, remember to provide meaningful `alt` text for images.